src/quantum/index.jsx

```diff
@@ -98,7 +98,7 @@ export default class QuantumIndex extends React.Component {
       },
       {
         title: 'Page Load tests (TP6)',
-        more: `/quantum/tp6?bits=${bits}`,
+        more: `/quantum/tp6?bits=${bits}&test=pageload`,
```

Comment:
https://deploy-preview-308--firefox-health-dashboard.netlify.com/quantum/tp6?bits=64&test=pageload shows:

![screen shot 2019-02-20 at 8 45 22 am](https://user-images.githubusercontent.com/3766511/53095910-eececd00-34eb-11e9-91a4-c5fd11b718c3.png)

Reply:
thanks for catching that!